safe/gis/vector/from_counts_to_ratios.py

```diff
@@ -69,19 +76,17 @@ def from_counts_to_ratios(layer, callback=None):
                     count_field=count_field['key']))
 
     if len(mapping) == 0:
-        # There is not a count field. Let's skip this layer.
+        # There is not a subset count field. Let's skip this layer.
         layer.commitChanges()
         return layer
 
```

Comment:
Can you add an information debug message about what we are doing here, with the textual formula?